source/Stargate-Model/AllowSpecifiedOriginsPolicy.class.st

```diff
@@ -11,17 +11,17 @@ Class {
 }
 
 { #category : #'instance creation' }
-AllowSpecifiedOriginsPolicy class >> onlyFrom: origins [
-	
-	^ self new initializeOnlyFrom: origins
+AllowSpecifiedOriginsPolicy class >> onlyFrom: anUrlCollection [
+
+	^ self new initializeOnlyFrom: ( anUrlCollection collect: [ :origin | origin asUrl origin ] )
```

Comment:
Why not extend `String` with a message like `asWebOrigin` and change the block to `anUrlCollection collect: #asWebOrigin`?